packages/components/src/tab-panel/index.tsx

```diff
@@ -58,14 +70,16 @@ export default function TabPanel( {
 		const newSelectedTab = find( tabs, { name: selected } );
 		if ( ! newSelectedTab ) {
 			setSelected(
-				initialTabName || ( tabs.length > 0 ? tabs[ 0 ].name : null )
+				initialTabName ||
+					( tabs.length > 0 ? tabs[ 0 ].name : undefined )
 			);
 		}
 	}, [ tabs ] );
 
 	return (
 		<div className={ className }>
 			<NavigableMenu
+				ref={ forwardedRef }
```

Comment:
Curious to why the `ref` was added to `NavigableMenu` and not the wrapper div ?

I'm asking because I wonder if consumers of the package would have a way to expect, in general, what component the `ref` holds (since we don't even document it in our docs)

Reply:
To be honest I'm pretty unclear about ref forwarding. Most examples I find online are cases where you want to do `myRef.focus()`. So I kind of understood it to mean that you want to pass the ref to the main interactive element 🤔 But I really don't know. Should make a non-ref-forwarding version of `WordPressComponent` asap? 😅

Reply:
Proposed in #43611